storage/rocksdb/src/transaction.rs

```diff
@@ -117,16 +115,17 @@ mod tests {
             input: vec![9, 9, 9].into(),
         };
         let signature = signer.sign_transaction_sync(&mut tx).unwrap();
-        let signed_tx = OpTxEnvelope::Eip1559(tx.into_signed(signature));
+        let signed_tx = tx.into_signed(signature);
+        let normalized_tx = NormalizedEthTransaction::try_from(signed_tx).unwrap();
 
         let transaction = ExtendedTransaction {
-            inner: signed_tx,
+            inner: normalized_tx.into(),
             block_number: 1,
             block_hash: B256::new(hex!(
                 "2222223123123121231231231231232222222231231231212312312312312322"
             )),
             transaction_index: 1,
-            effective_gas_price: 1,
+            effective_gas_price: U256::from(1),
```

Comment:
Nit: `U256` has a constant for 1 we could use instead of the conversion function.